src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
-
+Animal.alive = [];
 class Herbivore extends Animal {
-  // write your code here
+  hide(hidden = false) {
```

Comment:
```suggestion
  hide() {
```
Этот метод ничего не принимает, только инвертирует установленное значение